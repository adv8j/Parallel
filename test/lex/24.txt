int x = 5; // this is a comment using // // // "no error"
int y= 21;/ try this "error" /
int z=32; /   /

/**/

/*don't 
even
try*/

 /**//*/*/ /**/  // gives no error its correct

 /**************:)********/  //gives no error
 /*//////////////////////*********///////// 
 /    *     *     /  // error

