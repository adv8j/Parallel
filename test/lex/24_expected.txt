INT
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
single line comment
INT
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
DIV
IDENTIFIER
IDENTIFIER
STRING
DIV
INT
IDENTIFIER
ASSIGN
NUMBER
SEMICOLON
DIV
DIV
multi line comment
multi line comment
multi line comment
multi line comment
multi line comment
single line comment
multi line comment
single line comment
multi line comment
single line comment
DIV
MUL
MUL
DIV
single line comment
